README.md

```diff
@@ -0,0 +1,45 @@
+# Discovery based on Solid Social Agent WebId
+
+## Overview
+
+When an app allows a user to log in to a Solid Identity Provider, the app receives back an authenticated identifier for the user - their WebId.  The general purpose of this document is to describe what an app can discover based on the WebId.  More specific purposes are described below.
+
+A Social Agent - a person, group, or organization - may own one or more WebIds.  Each WebId is a URI that can be dereferenced to point to exactly one document, hereafter called a `profile document`.  The term 'profile document', as used here, means a `Social Agent WebId Document` and does not include WebId documents associated with Software Agents.  
+
+Profile documents may optionally point, through various means, to other documents whose intent is to identify the WebId owner and/or describe details about their data.  These other documents, whatever predicates are used to indicate them, and however they are named, will be referred to collectively as `extended profile documents`.  
+
+The term `profile` will be used here to refer to the data that can be gathered in the profile document and all extended profile documents it points to.
+
+Profiles contain two major types of information : 1) identifiying information such as a person or group's name, purpose, contacts,etc. and 2) organizing information such as what types of data the user has, where it is stored, and who has access to it. This document will focus on the latter - the information that an app can use to learn how to interact with the data associated with the Social Agent's WebId.
```

Comment:
```suggestion
Profiles contain two major types of information : 1) identifiying information such as a person or group's name, purpose, contacts, etc. and 2) organizing information such as what types of data the user has, where it is stored, and who has access to it. This document will focus on the latter - the information that an app can use to learn how to interact with the data associated with the Social Agent's WebId.
```

Reply:
Sorry, I can't spot the difference between the two.

Reply:
missing space before the `etc`. Your version has `,etc.`

Reply:
Ah, thanks!